File: android/app/src/main/java/l2/albitron/scrapyard/cloud/json/Notes.java
package l2.albitron.scrapyard.cloud.json;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;

@JsonPropertyOrder({"content", "format"})
public class Notes extends JSONEntity {
    @JsonProperty("content")
    public String content;
    @JsonProperty("format")
    public String format;
}

File: android/app/src/main/java/l2/albitron/scrapyard/cloud/db/model/Notes.kt
package l2.albitron.scrapyard.cloud.db.model

import com.fasterxml.jackson.annotation.JsonPropertyOrder
import com.fasterxml.jackson.annotation.JsonProperty

@JsonPropertyOrder("content", "format")
class Notes : JSONEntity() {
    @JsonProperty("content")
    var content: String? = null

    @JsonProperty("format")
    var format: String? = null
}